styles/style.css

```diff
@@ -0,0 +1,1028 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+font-size: 16px;
+display: flex;
+flex-direction: column;
+ width: 1020px;
+/* position: relative; */
+/* height: 100%; */
+position: absolute;
+left: 50%;
+transform: translateX(-50%);
+
+
+}
+
+
+ 
+ header {
+    background-color: #323232;
+    height: 63px; 
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    display: inline-block;
+   
+    
+} 
+
+header:after {
+content: "";
+width: 0px;
+height: 0px;
+border-style: solid;
+border-width: 26px 45px 0 45px;
+border-color: #323232 transparent transparent transparent;
+position: absolute;
+left: 50%;
+bottom: 0%;
+top: 63px;
+transform: translateX(-50%);
+
+}
+
+.navigation:before {
+    content: "";
+    border:  #e4e9eb 1px solid;
+    width: 410px ;
+    position: absolute;
+    left: 30px;
+    bottom: 0%;
+    top: 74px;
+    
+     }
+
+     
+.navigation::after {
+    content: "";
+    border: #e4e9eb  1px solid;
+    width: 410px ;
+    position: absolute;
+    right: 30px;
+    bottom: 0%;
+    top: 74px;
+    
+     }
+
+
+.menu {
+    position: relative;
+    left: 50%;
+    bottom: 0%;
+    /* top: 60px; */
+    transform: translateX(-50%);
+    text-align: justify;
```

Comment:
You can delete 
    **position: relative;
    left: 50%;
    bottom: 0%;
    /* top: 60px; */
    transform: translateX(-50%);**

Reply:
I thought I can use it in order to make menu in center horizontally

Reply:
You use flex, so it is unnecessary 
``` display: flex;
    justify-content: center;
```